ktor-features/ktor-websockets/jvm/src/io/ktor/websocket/WebSockets.kt

```diff
@@ -25,19 +25,26 @@ import kotlin.coroutines.*
  * }
  * ```
  *
- * @param pingIntervalMillis duration between pings or `null` to disable pings
- * @param timeoutMillis write/ping timeout after that a connection will be closed
- * @param maxFrameSize maximum frame that could be received or sent
- * @param masking whether masking need to be enabled (useful for security)
+ * @param pingIntervalMillis duration between pings or `null` to disable pings.
+ * @param timeoutMillis write/ping timeout after that a connection will be closed.
+ * @param maxFrameSize maximum frame that could be received or sent.
+ * @param masking whether masking need to be enabled (useful for security).
+ * @param extensionsConfig is configuration for WebSocket extensions.
  */
-public class WebSockets(
+public class WebSockets @OptIn(ExperimentalWebSocketExtensionApi::class) constructor(
```

Comment:
Same question here: looks like we should propagate the annotation instead of opt-in

Reply:
Yep. In this constructor we should, in old we shouldn't